R/fda-table_33.R

```diff
@@ -0,0 +1,60 @@
+# Pre-processing
+
+library(tidyverse)
```

Comment:
One last suggestion - we don't need to load all `tidyverse` packages, just `dplyr` should be sufficient.

```suggestion
library(dplyr)
```